css-nesting-1/Overview.bs

```diff
@@ -270,14 +270,14 @@ The Nesting At-Rule: ''@nest'' {#at-nest}
 	Some valid nesting selectors,
 	like ''.foo &'',
 	are disallowed,
-	and editting the selector in certain ways can make the rule invalid unexpectedly.
+	and editing the selector in certain ways can make the rule invalid unexpectedly.
 	As well,
-	some people find the nesting difficult to visually distinguish
+	some people find the nesting challenging to distinguish visually
```

Comment:
This line wasn't wrong, but the "correction" isn't wrong either, so I'll go ahead and accept anyway. ^_^

Reply:
Thank you! 👌